pkg/maps/lbmap/lbmap.go

```diff
@@ -49,7 +49,7 @@ type LBBPFMap struct{}
 func (*LBBPFMap) UpsertService(
 	svcID uint16, svcIP net.IP, svcPort uint16,
 	backendIDs []uint16, prevBackendCount int,
-	ipv6 bool, svcType loadbalancer.SVCType) error {
```

Comment:
You will need to update https://github.com/cilium/cilium/blob/c6e876c1ac568db853af21ccde5445cef9da1a5e/pkg/maps/lbmap/lbmap_mock.go#L36-L37 as well.